judgels-backends/judgels-commons/judgels-service-core/src/main/java/judgels/service/ServiceUtils.java

```diff
@@ -85,8 +87,9 @@ public static Response buildImageResponseFromText(String text, Date lastModified
         g2d.fillRect(0, 0, img.getWidth(), img.getHeight());
         g2d.setColor(Color.BLACK);
         int nextLinePosition = margin;
-        for (String s : textList) {
-            g2d.drawString(s.replaceAll("\t", "    "), 0, nextLinePosition);
+        for (int i = 0; i < textList.length; i++) {
+            String s = textList[i];
+            g2d.drawString(String.format(lineTemplate, i+1, s.replaceAll("\t", "    ")), 0, nextLinePosition);
```

Comment:
```suggestion
            g2d.drawString(String.format(lineTemplate, i + 1, s.replaceAll("\t", "    ")), 0, nextLinePosition);
```

(not sure why it was not caught by checkstyle)

Reply:
I forgot to install Checkstyle IDEA, hahahahha